src/core_plugins/metrics/public/components/_vis_picker.scss

```diff
@@ -0,0 +1,4 @@
+.tvbVisPickerItem {
```

Comment:
I think this can be removed? Looks like it just overwrites the tab sizing?

Is this what got solved later here? https://github.com/elastic/eui/pull/1244

Reply:
Not yet, Kibana is only on 4.5.1, and that went in at 4.6

Reply:
Plus I'm adding some boldness too because it gets lost in the sea of tabs.